sos/report/plugins/fail2ban.py

```diff
@@ -0,0 +1,40 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, DebianPlugin, UbuntuPlugin
+
+
+class Fail2Ban(Plugin, DebianPlugin, UbuntuPlugin):
+
+    short_desc = "Fail2Ban daemon"
+
+    plugin_name = "fail2ban"
+    profiles = ('security',)
+    packages = ('fail2ban',)
```

Comment:
This package exists on RHEL, but so does `fail2ban-server` , which can be installed standalone.

Reply:
Added.